Makefile

```diff
@@ -63,7 +63,7 @@ install-column-apt:
 
 .PHONY: generate-deepcopy
 generate-deepcopy: "$(MYGOBIN)/deepcopy-gen"
-	hack/run-in-gopath.sh deepcopy-gen --input-dirs ./pkg/apis/... -O zz_generated.deepcopy --go-header-file ./LICENSE_TEMPLATE_GO
+	hack/run-in-gopath.sh deepcopy-gen --input-dirs ./pkg/apis/actuation/ -O zz_generated.deepcopy --go-header-file ./LICENSE_TEMPLATE_GO
```

Comment:
is this needed for this PR? It can probably be pulled out.

Reply:
No longer needed.